### PR TITLE
MB-11880 Add history log for when MilMove deprecates a old payment request

### PR DESCRIPTION
**Pull request by YanZ777**
## Summary
Added a new event type for when a payment request gets deprecated.

I spoke with Julie, and she agreed to have the status be labeled text details, which is why it says `Status: Deprecated` instead of `Deprecated payment request ####-####-#`

[MB-11408](https://dp3.atlassian.net/browse/MB-11408), which Kenneth is working on, does overlap with mine. After discussing it with him, we both decided to have my changes merged first, with his being a follow up.

@michaelvaldes-truss I renamed everything to `action` instead of `actions` inside of `moveHistoryTemplate.js` since I am under the impression that was a mistaken change in [this PR](https://github.com/transcom/mymove/pull/8574/files). Let me know if that assumption is incorrect.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make office_client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the office app
2. Login as TIO
3. Click on a move
4. Ensure there’s a payment request
5. Click on Review weights button
6. Edit a shipment weight so that it falls below a little below what its original numbers are listed
7.  Go back to the Payment Requests tab and the payment request should be marked as deprecated

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output

## Screenshots
image

### CODE REVIEW

**Review by juliemoon — APPROVED** [review]
This looks great, thanks Yan!

File: src/constants/moveHistoryEventTemplate.js [hunk 1]
```diff
@@ -390,16 +391,32 @@ export const updateMTOShipmentAgentEvent = buildMoveHistoryEventTemplate({
...
+  getDetailsLabeledDetails: ({ changedValues }) => {
+    const { status } = changedValues;
+    const formattedValues = { ...changedValues };
+    if (status) {
+      formattedValues.status = PAYMENT_REQUEST_STATUS_LABELS[status];
+    }
+    return formattedValues;
+  },
+});
+
```
Comment (donaldthai) [inline src/constants/moveHistoryEventTemplate.js:409]: question: Noticed for some events regarding the payment request table some are using `getDetailsLabeledDetails` and others are using `getStatusDetails`. Just wondering if we wanted to use the `getDetailsLabeledDetails` for a particular reason?

**Review by michaelvaldes-truss — APPROVED** [review]
**praise:** thanks for catching the typos!

File: src/constants/moveHistoryEventTemplate.js (hunk 1 above)
Comment (YanZ777) [reply src/constants/moveHistoryEventTemplate.js]: I did not notice the `getStatusDetails`, but that is probably a better fit than `getDetailsLabeledDetails`. It has been changed it to use that.

**Pull request merged**